src/components/sync-configurator/IndicatorSync.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import BaseSyncConfigurator from "./BaseSyncConfigurator";
+import PropTypes from "prop-types";
+import { IndicatorModel } from "../../models/d2Model";
+
+export default class IndicatorSync extends React.Component {
```

Comment:
Well, we could try to make it generic too by adding the model in the Routes. The only one I think it will be really different is OU if we add a tree filter. The problem then is that we are coupling too much responsibility in the route side.

Reply:
Agreed, we want the routes file as dumb as possible, let's keep as it is.